Formula/o/opentdf-cli.rb

```diff
@@ -0,0 +1,52 @@
+class OpentdfCli < Formula
+  desc "OpenTDF command-line interface"
+  homepage "https://docs.opentdf.io/"
+  url "https://github.com/opentdf/otdfctl.git",
+      tag:      "v0.11.4",
+      revision: "1e64065d08547658e436dc167b108c2a6e9c0ccd"
+  license "BSD-3-Clause-Clear"
+  head "https://github.com/opentdf/otdfctl.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_releases
+  end
+
+  keg_only :versioned_formula
```

Comment:
Why?

Reply:
I was following the kubectl formula per the Homebrew's advice to find a similar formula. I'll remove all the questionable parts.